packages/react-server/src/ReactFlightServer.js

```diff
@@ -1328,6 +1382,10 @@ function retryTask(request: Request, task: Task): void {
       x.then(ping, ping);
       task.thenableState = getThenableStateAfterSuspending();
       return;
+    } else if (enablePostpone && x.$$typeof === REACT_POSTPONE_TYPE) {
+      const postponeInstance: Postpone = (x: any);
+      logPostpone(request, postponeInstance.message);
+      emitPostponeChunk(request, task.id, postponeInstance);
```

Comment:
Is the task supposed to be in a new status here? I don't see why it would get retried but if it did it would attempt to resolve the model again.

Reply:
yea, that's right and it should probably be removed from the abortable tasks list so it doesn't get aborted later.